authors/apps/authentication/views.py

```diff
@@ -128,3 +150,22 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+class UserActivateAPIView(APIView):
+    """
+    Activate account using the link sent to the user's email. 
+
+    Decodes the token in the url and confirms whether the user 
+    is in the database using the username in the token.
+    If successful, the user is activated.
+    """
+
+    def get(self, request, token):
+        try:
+            data = jwt.decode(token, settings.SECRET_KEY)
+            user = User.objects.get(username=data['username'])
+        except:
+            return HttpResponse('Activation link is invalid!')
+        user.is_active = True
+        user.save()
+        return HttpResponse("Account was verified successfully")
```

Comment:
@Bileonaire This one too, it will be a more valuable output as json

Reply:
Okay. let me sort that out